### ISSUE TITLE
Anycubic Kobra family

### ISSUE THREAD

**Issue by dlico23** [prusa3d/PrusaSlicer-settings#173]
Is there any hope that it will be added Anycubic Kobra family?

**Comment by rtyr** [comment]
Yes, but 3rd party printer profiles are mostly about community contributions.

**Comment by ifohancroft** [comment]
#250 It's not the full Kobra family, but it's all I have provided by Anycubic on the microSD of my Kobra 2 for the Kobra family for PrusaSlicer. Altered of course, to follow the conventions of this repo.
